lib/services/pages.js

```diff
@@ -282,6 +282,8 @@ function publish(uri, data, locals) {
         // make public
         addOpToMakePublic(ops, prefix, publicUrl, uri);
 
+        pageData.lastModified = new Date().toISOString();
```

Comment:
Do we want this in amphora? The publishing chain could handle it in the sites instance.